modules/ROOT/pages/_partials/mmp-deploy-to-cloudhub-2.adoc

```diff
@@ -233,6 +233,8 @@ Configuration example:
 
 | `pathRewrite` | Supplies the base path expected by the HTTP listener in your application. This value must begin with `/`. This parameter is used only for applications that are deployed to xref:cloudhub-2::ch2-private-space-about.adoc[private space]. | No
 
+| `instanceType` | Instance type. Only UBP organizations can use this flag. Non-UBP organizations use the `--replicaSize` flag. See xref:general::pricing.adoc[]. | No
```

Comment:
replicaSize is not avaliable in MMP, non-UBP customers  should use vCores (that are documented in this page)